README.md

```diff
@@ -61,62 +65,61 @@ rest of the framework. This may be the most appealing design goal, compared to o
 
 ### Love It or Leave It
 
-HTTP is focused on accomplishing the established goals, and may not be the right fit for all
-applications. So the rest of the core Angular2 framework should not give special treatment to HTTP,
+Http is focused on accomplishing the established goals, and may not be the right fit for all
+applications. So the rest of the core Angular2 framework should not give special treatment to Http,
 so that developers can easily use the http library of their choice. Perhaps other parts of a broader
-Angular2 data framework would be built with this HTTP implementation in mind, but it should be
+Angular2 data framework would be built with this Http implementation in mind, but it should be
 possible to bridge other http libraries.
 
 ## Design Anti-Goals
 
 ### Only Understand Response Types
 
-HTTP will not concern itself with the structure of data being sent or returned; it will only care
+Http will not concern itself with the structure of data being sent or returned; it will only care
```

Comment:
Good question. My thought was user-specified response type would essentially proxy a user-specified responseType to the underlying connection, but perhaps the library should be smarter.